view/rooms/app.js

```diff
@@ -83,6 +89,35 @@ function createHeader(bot) {
   return section;
 }
 
+/**
+ * Creates footers for each bot added to the UI
+ *
+ * @param {Object} bot - Bot response with UI
+ * @returns {DOM} footer - Footer section
+ */
+function createFooter(bot) {
+  const footer = document.createElement('h3');
+  const linkedElement = document.createElement('a');
+  const gitHubImage = document.createElement('img');
+  const gitHubLogo = 'https://image.flaticon.com/icons/svg/25/25231.svg';
```

Comment:
I'd use more reliable source for this logo. E.g. I'd consider github.com to be more reliable - somewhere in https://github.com/logos